docs/modules/ROOT/pages/sentinel.adoc

```diff
@@ -349,29 +349,30 @@ NOTE: Only alerts that match other conditions (Severity, Alert IDs) will invoke
 
 The request body will contain the following structure. 
 
+NOTE: We have updated the Forta Alert schema in correspondence with the new https://docs.forta.network/en/latest/api/[Forta API]. The following changes were made: `alert_id` -> `alertId`, `scanner_count` -> `scanNodeCount`, `type` -> `findingType`, `tx_hash` -> `transactionHash`, `chain_Id` -> `chainId`, Agent `name` removed. Old properties are now deprecated but we will continue to send both to remain backwards compatible.
```

Comment:
add the date of release for this change, so people can assess whether they need to care about this